match-standards-serum-mix01.Rmd

```diff
@@ -5,44 +5,22 @@ output:
   rmarkdown::html_document:
     highlight: pygments
     toc: true
+    toc_float: true
     toc_depth: 3
     fig_width: 5
 ---
 
-```{r libraries, message = FALSE, warning = FALSE, echo = FALSE}
-library(xcms)
-library(pander)
-library(MetaboCoreUtils)
-library(MsCoreUtils)
-library(MetaboAnnotation)
-library(BiocParallel)
-library(Spectra)
-library(RColorBrewer)
-library(MetaboAnnotation)
-library(pheatmap)
-library(MsFeatures)
-setMSnbaseFastLoad(FALSE)
-```
-
-```{r general-settings, echo = FALSE, message = FALSE}
-mix <- 1
-mix_name <- paste0("Mix", ifelse(mix < 10, paste0(0, mix), mix)) 
-IMAGE_PATH <- paste0("images/match-standards-serum-", tolower(mix_name),"/")
-if (dir.exists(IMAGE_PATH)) unlink(IMAGE_PATH, recursive = TRUE)
-RDATA_PATH <- paste0("data/RData/match-standards-serum-", tolower(mix_name), "/")
-dir.create(IMAGE_PATH, showWarnings = FALSE, recursive = TRUE)
-dir.create(RDATA_PATH, showWarnings = FALSE, recursive = TRUE)
-
-# Define the mzML files *base* path (/data/massspec/mzML/ on the cluster)
-MZML_PATH <- "~/mix01/"
-# MZML_PATH <- "/data/massspec/mzML/"
-
-register(SerialParam())
 
-library(knitr)
-opts_chunk$set(message = FALSE, warning = FALSE, fig.width = 10, fig.height = 8,
-               dev = c("png", "pdf"), fig.path = IMAGE_PATH)
+```{r, include = FALSE, cached = FALSE}
+knitr::read_chunk("R/match-standards-chunks.R")
+MIX <- 1
+MATRIX <- "Serum"
+POLARITY <- "POS"
+```
 
+```{r, libraries, echo = FALSE, message = FALSE}
```

Comment:
this code block will be replaced by `rmarkdown::render` with the code block defined in the *match-standards-chunks.R* file named `## ---- libraries ----`